src/recenter/directive.js

```diff
@@ -1,7 +1,15 @@
-goog.provide('ngeo.recenterDirective');
+goog.provide('ngeo.recenter.directive');
```

Comment:
Should be in `src/map/recenter.js`. @sbrunner, do you want to add a note in https://jira.camptocamp.com/browse/GSGMF-172 explaining the files need to be renamed (not moved in a `directive` subdirectory)?

Then the symbol should be `ngeo.map.recenter`. (I would drop the `directive` suffix).

Reply:
+1